build.sbt

```diff
@@ -22,20 +22,21 @@ initialize := {
   initialize.value
 }
 
-// akka.AkkaBuild.buildSettings
+akka.AkkaBuild.buildSettings
 shellPrompt := { s => Project.extract(s).currentProject.id + " > " }
 resolverSettings
 
+def isScala211: Boolean = System.getProperty("akka.build.scalaVersion", "").startsWith("2.11")
+
 // When this is updated the set of modules in ActorSystem.allModules should also be updated
-lazy val aggregatedProjects: Seq[ProjectReference] = Seq(
+lazy val aggregatedProjects: Seq[ProjectReference] = List[ProjectReference](
```

Comment:
Just curious why the change to `List` explicitly

Reply:
There is some magic implicits between Project and ProjectReference and it didn't like when I broke it up in two lists. Maybe it could still be Seq with the additional type annotations, but this works.